keylime-agent/src/keys_handler.rs

```diff
@@ -865,9 +873,9 @@ mod tests {
             rsa_oaep_encrypt(&quotedata.pub_key, u.as_ref()).unwrap(); //#[allow_ci]
 
         let ukey = KeylimeUKey {
-            encrypted_key: base64::encode(&encrypted_key),
+            encrypted_key: general_purpose::STANDARD.encode(&encrypted_key),
             auth_tag: hex::encode(auth_tag),
-            payload: payload.map(base64::encode),
+            payload: payload.map(general_purpose::STANDARD.encode),
```

Comment:
There is a complain here, should be something like `payload.map(|p| general_purpose::STANDARD.encode(p.as_ref())),`, as before?

Reply:
Yes, probably that will be the way to go. Thanks for the suggestion!